src/styles/blocks/page.scss

```diff
@@ -0,0 +1,27 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+.main {
```

Comment:
<img width="660" alt="image" src="https://user-images.githubusercontent.com/102411130/209311868-907aad1b-500f-4cba-bcce-89268689465a.png">

Reply:
file should have the same name as block